python/pyspark/sql/tests.py

```diff
@@ -6323,6 +6333,33 @@ def ordered_window(self):
     def unpartitioned_window(self):
         return Window.partitionBy()
 
+    @property
+    def sliding_row_window(self):
```

Comment:
do these need to be properties or could they be variables?

Reply:
I think they could be variables. I don't particular care one way or the other.